common/lib/common-utils/src/bufferBrowser.ts

```diff
@@ -78,6 +88,12 @@ export class IsoBuffer extends Uint8Array {
     static from(value, encodingOrOffset?, length?): IsoBuffer {
         if (typeof value === "string") {
             return IsoBuffer.fromString(value, encodingOrOffset as string | undefined);
+        // Capture any typed arrays, including Uint8Array (and thus - IsoBuffer!)
+        } else if (value !== null && typeof value === "object" && isArrayBuffer(value.buffer)) {
+            // Support currently for full array, no view ports! (though it can be added in future)
+            assert(value.byteOffset === 0);
```

Comment:
we assert byteOffset  ==0 but above in isArrayBuffer we require it to be undefined

Reply:
value.buffer.byteOffset  == undefined.
value.byteOffset === 0

Reply:
Basically we say outer is typed array, inner is not

Reply:
oh. i see. we are looking a value.buffer above, not value itself